CMakeLists.txt

```diff
@@ -972,6 +972,7 @@ if(SDL_LIBC)
     memory.h
     signal.h
     stdarg.h
+    stdbool.h
```

Comment:
Perhaps add `HAVE_STD_BOOL_H` to cmake/PreseedMSVCCache.cmake and cmake/PreseedEmscripenCache.cmake.

Reply:
Done!